lms/djangoapps/teams/views.py

```diff
@@ -366,7 +366,14 @@ def get(self, request):
                 return Response(error, status=status.HTTP_400_BAD_REQUEST)
             result_filter.update({'topic_id': topic_id})
         if text_search and CourseTeamIndexer.search_is_enabled():
-            search_engine = CourseTeamIndexer.engine()
+            try:
+                search_engine = CourseTeamIndexer.engine()
+            except ElasticSearchConnectionError:
+                return Response(
+                    build_api_error(ugettext_noop('Error connecting to elasticsearch')),
+                    status=status.HTTP_400_BAD_REQUEST
```

Comment:
Nit: I usually think of a bad request as being a problem with the client, so I'd think it should be a 5xx error. Should it maybe be a 503?

http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html